smartnotes/settings.py

```diff
@@ -58,7 +58,9 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [
+            BASE_DIR / "static/templates",
```

Comment:
Why?

Reply:
- the static folder, is used to store static files such as css, html and js etc.
- Also it allows for abstraction common html template and inject only the special content of each templates of the app

Reply:
By default django knows where to look for template files, IIRC

Reply:
I think it should be told, when isolated to a static folder